pages/team.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: page
+title: MDAnalysis team
+---
+
+MDAnalysis is a community-driven project that is made possible through the efforts of many members who contribute in numerous and diverse ways, ranging from direct package development, maintenance, documentation, communication, and managerial responsibilities. On this page we list identified project roles and those historically responsible for the tasks. In the future, we plan to also list the contributors currently responsible for overseeing and managing associated tasks. We note that the listed roles on this page can differ significantly in scope and required effort.
+
+## Roles
+
+In this section we list contributors who have led or contributed substantial effort to the roles listed below.
+At present, we have only listed leading figures from the past 2-3 years, i.e. from 2020 onwards.
+However, we hope to eventually acknowledge everyone who has been responsible for a particular role throughout
+the lifetime of the project.
```

Comment:
```suggestion
the lifetime of the project.

This organisation in teams started in the fall of 2023. Many people contributed to these tasks before then and may not appear in the table.
```

Reply:
Thanks for the suggestion - I do think this is a good clarification to make; but leaving for now since I agree with Jenna's comment below to move this a little higher in the text

Reply:
It's now been added to the text!